chap-architecture.tex

```diff
@@ -1616,12 +1616,11 @@ \section{Capability-Aware Instructions}
 \item[Manipulate capability fields]
 These instructions modify capability-register fields, setting them to values
 moved from integer registers, subject to constraints such as monotonicity and
-representability: \insnref{CAndAddr}, \insnref{CAndPerm},
-\insnref{CClearTag},
+representability: \insnref{CAndPerm}, \insnref{CClearTag},
 \insnref{CIncOffset}, \insnref{CIncOffsetImm},
 \insnref{CSetAddr}, \insnref{CSetBounds},
 \insnref{CSetBoundsExact}, \insnref{CSetBoundsImm},
-\insnref{CSetFlags}, and
+\insnref{CSetFlags}, \insnref{CSetHigh}, and
```

Comment:
Add CSetHigh here appears to be an unrelated change?

Reply:
Ah yeah, I replaced the no longer valid instruction with the newly added modification one. Will update to just remove the CAndAddr.